bootstrap/namespaces/helmfile/values/namespaces-sc.yaml.gotmpl

```diff
@@ -1,27 +1,106 @@
 namespaces:
 - name: cert-manager
+  labels:
+    pod-security.kubernetes.io/audit: baseline
+    pod-security.kubernetes.io/audit-version: v1.25
```

Comment:
Would it be beneficial to be able to configure the PSA versions? I'm thinking we need these changes before we upgrade from v1.24.

Reply:
I have not looked that much how these labels work. But it sounds reasonable

Reply:
@robinelastisys as you did some testing with these labels you maybe have some input?

Reply:
Apparently v1.25 works on v1.24. :thinking:

Reply:
The version label is apparently optional, defualt is the current kubernetes verison. I think we can remove it. 
Also i think we want to keep the labels `warn` and `audit`
```
Pods are often created indirectly, by creating a workload object such as a Deployment or Job. The workload object defines a Pod template and a controller for the workload resource creates Pods based on that template. To help catch violations early, both the audit and warning modes are applied to the workload resources. However, enforce mode is not applied to workload resources, only to the resulting pod objects.
```

Reply:
I removed that version label